jupyter_notebook/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM jupyter/scipy-notebook
```

Comment:
So this is what builds unai/scipy-notebook-extended but where does it get added under that as an image name somewhere?

Reply:
It is built as part of the manual deployment process. The readme outlines how to build the image.